### PR TITLE
chore: update READMEs

### PR DESCRIPTION
**Pull request by ascorbic**
### Summary

Creates READMEs for the individual packages, and remove some deprecated parts from the top-level README

### Standard checks:

- [ ] Check the Deploy Preview's Demo site for your PR's functionality
- [ ] Add docs when necessary

---

🧪 Once merged, make sure to update the version if needed and that it was published correctly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,19 +2,8 @@
...
 
-<p align="center">
-  <a aria-label="npm version" href="https://www.npmjs.com/package/@netlify/plugin-nextjs">
-    <img alt="" src="https://img.shields.io/npm/v/@netlify/plugin-nextjs">
-  </a>
-  <a aria-label="MIT License" href="https://img.shields.io/npm/l/@netlify/plugin-nextjs">
-    <img alt="" src="https://img.shields.io/npm/l/@netlify/plugin-nextjs">
-  </a>
-</p>
-
```
Comment (ascorbic) [inline README.md:13]: Moved to the plugin package

File: README.md [hunk 2]
```diff
@@ -55,21 +44,9 @@ by targeting the `/_next/image/*` route:
...
-variable `NEXT_USE_NETLIFY_EDGE` to `true`.
-
-### No nested middleware in Next 12.2.0
-
-In Next 12.2.0, nested middleware [has been deprecated](https://nextjs.org/docs/messages/middleware-upgrade-guide) in
-favor of root level middleware. If you are not using edge functions then this means that you won't get the benefits of
-using a CDN, and ISR will not work.
-
-To fix this issue, you can run your middleware on [Netlify Edge Functions](#netlify-edge-functions) by setting the
-environment variable `NEXT_USE_NETLIFY_EDGE` to `true`.
```
Comment (ascorbic) [inline README.md:72]: Edge is now default

File: packages/next/README.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+before they are sent to the browser. This allows you to personalize pages on the fly, even if they are
+statically-generated. It includes baked-in support for:
+
+- HTML rewrites
+- Page data transforms
+- Modifying request headers
+- Access to response body
+
+For full details,
+[see the docs](https://docs.netlify.com/integrations/frameworks/next-js/middleware/#next-js-advanced-middleware-with-the-netlify-next-library).
```
Comment (ascorbic) [inline packages/next/README.md:30]: The goal for this is mostly just to give context and direct people to the docs

File: packages/runtime/README.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+  <a aria-label="npm version" href="https://www.npmjs.com/package/@netlify/plugin-nextjs">
+    <img alt="" src="https://img.shields.io/npm/v/@netlify/plugin-nextjs">
+  </a>
+  <a aria-label="MIT License" href="https://img.shields.io/npm/l/@netlify/plugin-nextjs">
+    <img alt="" src="https://img.shields.io/npm/l/@netlify/plugin-nextjs">
+  </a>
+</p>
+
+This package handles the build process for Next.js sites on Netlify. You should not normally need to install it
+yourself, as it is used automatically during builds of Next.js sites.
```
Comment (ascorbic) [inline packages/runtime/README.md:15]: Just for context.

File: README.md [hunk 5]
```diff
@@ -55,21 +44,9 @@ by targeting the `/_next/image/*` route:
...
+Next.js Middleware works out of the box on Netlify. By default, middleware runs using Netlify Edge Functions. For legacy
+support for runnign Middleware at the origin, set the environment variable `NEXT_DISABLE_NETLIFY_EDGE` to `true`. Be
```
Comment (whitep4nth3r) [inline README.md:48]: Typo `runnign`

File: README.md [hunk 6]
```diff
@@ -127,7 +104,7 @@ package = "@netlify/plugin-nextjs"
 
 ## Manually upgrading from an older version of the Next.js Runtime
 
-If you previously set these values, they're no longer needed and can be removed:
+If you previously set these values, they're no longer needed and should be removed:
 
 - `distDir` in your `next.config.js`
 - `node_bundler = "esbuild"` in `netlify.toml`
```
Comment (whitep4nth3r) [inline README.md:110]: What about NEXT_USE_NETLIFY_EDGE in the toml?

File: packages/next/README.md [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+  </a>
+</p>
+
+# `@netlify/next`
+
+The `@netlify/next` package makes Next.js even better.
+
+## Next.js Advanced Middleware
+
+Regular Next.js Middleware doesn’t provide access to the actual response, or allow you to modify the request. The
```
Comment (whitep4nth3r) [inline packages/next/README.md:18]: Change to "It gives full access to the request and response objects"

Comment (whitep4nth3r) [reply packages/next/README.md:18]: Sorry this tagged the wrong line, but it's about adding "to the" in that sentence

File: packages/next/README.md [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+`@netlify/next` library brings the power of Netlify Edge Functions to Next.js Middleware. It gives full access request
```
Comment (orinokai) [inline packages/next/README.md:19]: [pebble] It gives full access *to* request and response objects

File: README.md (hunk 6 above)
Comment (ascorbic) [reply README.md:110]: Ah, good addition

**Review by orinokai — APPROVED** [review]
Just a little typo

**Review by orinokai — APPROVED** [review]

**Pull request merged**